uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/helpers/SessionHelpersTrait.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_SessionHelpersTrait {
+
+  private function registerCurrentLoggedInContactInSession($contactID) {
```

Comment:
can you please look for other classes that have their own implementation of this and replace it with the trait?